README.md

```diff
@@ -118,8 +118,7 @@ storageConfig:
   ```
 For configuration and options, see the [expanded overview](./docs/overview.md) and [usage](./docs/usage.md) docs.
 
-<sup>1</sup> For this example, the `create` and `publish` steps are run on the same machine. Therefore your `~/.docker/config.json`
-should contain auth config for both release/catalog source images _and_ your mirror registry.
+<sup>1</sup> For this example, the `create` and `publish` steps are run on the same machine. Therefore your `~/.docker/config.json` or `${XDG_RUNTIME_DIR}/containers/auth. json` should contain auth config for both release/catalog source images _and_ your mirror registry.
```

Comment:
There is an unintentional white space between `auth.` and `json` I think. Or maybe this is intentional?

Reply:
Copy and paste error :) Thanks for catching it.